docs/our-approach-to-data.md

```diff
@@ -332,9 +332,8 @@ export const items = createReducer( defaultState, {
 	[DESERIALIZE]: state => fromJS( state )
 } );
 ```
-If your reducer state is already a plain object, you may choose to omit `SERIALIZE` and `DESERIALIZE` handlers, or 
-simply define them as returning the current state. However, please note that the subtree can still see errors from 
-changing data shapes, as described below.
+If your reducer state is already a plain object, the `SERIALIZE` and `DESERIALIZE` handlers are not needed.
+However, please note that the subtree can still see errors from changing data shapes, as described below.
```

Comment:
For simplicity I'd recommend that anything that we persist should have a schema. In theory we might be ok if we happened to add a schema after we make the first data change, but it's unlikely to happen.